docs-ref-conceptual/azure-cli-reference-for-IoT.md

```diff
@@ -37,57 +37,49 @@ The IoT **extension** introduces rich features and functionality to manage, mani
 
 | Reference | Has core | Description
```

Comment:
The core and extension tables need to be combined as we no longer have separate lists.  The new TOC is not "extension type" aware and it doesn't make sense to provide two links to the same place.

Reply:
Hi @dbradish-microsoft, what does "once table format request for IoT" mean?